api/service.py

```
@@ -593,6 +653,93 @@ async def get_anomaly_history(
+    return UserResponse(
+        id=user.id,
+        username=user.username,
+        role=user.role.value,
+        email=user.email,
+        created_at=user.created_at,
+        is_active=user.is_active
+    )
+
+
+@app.get("/api/v1/auth/users/me", response_model=UserResponse)
+async def get_current_user_info(current_user: User = Depends(get_current_user)):
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)